.circleci/config.yml

```diff
@@ -493,7 +493,7 @@ jobs:
       - run:
           name: 'Install DBI.pm'
           command: |
-            apt-get update && apt-get install libdbi-perl && apt-get install libdbd-pg-perl
+            apt-get update && apt-get install libdbi-perl && apt-get install libdbd-pg-perl postgresql-<< parameters.pg_major >>-wal2json
```

Comment:
this should be part of our images

Reply:
Updated the dependencies  in a new build from PR:
https://github.com/citusdata/the-process/pull/121